helm-charts/FATE/templates/core/client/statefulSet.yaml

```diff
@@ -43,8 +43,13 @@ spec:
               value: "9380"
             - name: FATE_SERVING_HOST
               value: "{{.Values.modules.serving.ip}}:{{.Values.modules.serving.port}}"
+            - name: NOTEBOOK_HASHED_PASSWORD
+              value: {{ .Values.modules.client.notebook_hashed_password }}
+            - name: PORT
```

Comment:
Are we using this?

Reply:
No, I found this extra env which is used for testing. I have submittd a patch to remove it few minites ago.